spring-kafka/src/main/java/org/springframework/kafka/listener/KafkaMessageListenerContainer.java

```diff
@@ -203,8 +205,6 @@ protected void doStop() {
 
 	private class ListenerConsumer implements SchedulingAwareRunnable {
 
-		private final Log logger = LogFactory.getLog(this.getClass());
-
 		private final CommitCallback callback = new CommitCallback();
```

Comment:
I added a local logger explicitly to avoid referencing the enclosing class on each call.